app/assets/javascripts/components/Header.js

```diff
@@ -0,0 +1,63 @@
+define(['jquery', 'DoughBaseComponent', 'utilities'], function($, DoughBaseComponent, utilities) {
+  'use strict';
+
+  var Header,
+      defaultConfig = {};
+
+  Header = function($el, config) {
+    Header.baseConstructor.call(this, $el, config, defaultConfig);
+
+    this.activeClass = 'mobile-header__search-active';
+    this.$header = $(document).find('[data-site-header]');
```

Comment:
Better to refer to $el here for the header component, so either this.$header = $el or use this.$el throughout.

Reply:
2c86e28